iOS/baseball/baseball/ViewController/ScoreBoardViewController.swift

```diff
@@ -0,0 +1,99 @@
+        self.setTeamControllerBarTitle()
+        self.tableViewCellRegisterNib()
+        self.itemListDidLoad()
+        self.tableViewHeight.constant = self.playerScoreTableView.contentSize.height
+    }
+    
+    private func setTeamControllerBarTitle() {
+        teamControllBar.setTitle("A~", forSegmentAt: 0)
+        teamControllBar.setTitle("B~", forSegmentAt: 1)
+    }
+    
+    private func tableViewCellRegisterNib() {
+        let nib = UINib(nibName: PlayerScoreTableViewCell.identifier, bundle: nil)
+        self.playerScoreTableView.register(nib, forCellReuseIdentifier: PlayerScoreTableViewCell.identifier)
+
+    }
+    
+    private func itemListDidLoad() {
+        var snapshot = Snapshot()
+        let playerScoreBoards = [
+            PlayerScoreBoard(id: 1, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 2, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 3, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 4, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 5, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 6, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 7, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 8, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 9, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 10, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 11, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 12, name: "정대현", TPA: 1, hits: 3, out: 1),
+            PlayerScoreBoard(id: 13, name: "양원종", TPA: 1, hits: 1, out: 2),
+            PlayerScoreBoard(id: 14, name: "정대현", TPA: 1, hits: 3, out: 1),
+        ]
+        snapshot.appendSections(["test"])
+        snapshot.appendItems(playerScoreBoards, toSection: "test")
+        dataSource.apply(snapshot)
+    }
+    
+    private func makeDataSource() -> DataSource {
+        return DataSource(tableView: self.playerScoreTableView) { (tableView, indexPath, playerScoreBoard) -> PlayerScoreTableViewCell? in
+            guard let cell = tableView.dequeueReusableCell(withIdentifier: PlayerScoreTableViewCell.identifier, for: indexPath) as? PlayerScoreTableViewCell else { return PlayerScoreTableViewCell() }
+            cell.configure(playerScoreBoard: playerScoreBoard)
+            return cell
+        }
+    }
+}
+
+extension ScoreBoardViewController {
+    private func sumScore(of scores: [UILabel]) -> String{
+        return "\(scores.reduce(0, { $0 + (Int($1.text ?? "0") ?? 0)  }))"
+    }
+}
+
+extension ScoreBoardViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let header = Bundle.main.loadNibNamed(PlayerScoreHeaderTableViewCell.identifier, owner: self, options: nil)?.last as! PlayerScoreHeaderTableViewCell
```

Comment:
강제 언래핑은 분명하게 사용해야 할 때 있습니다. 하지만 앱 크래시 원인이 되기 때문에 정말 필요한 곳인지 생각하고 사용하세요!

```suggestion
       guard let header = Bundle.main.loadNibNamed(PlayerScoreHeaderTableViewCell.identifier, owner: self, options: nil)?.last as? PlayerScoreHeaderTableViewCell else {
           // ....
```